### PR TITLE
create ThrownExpBottle entity

### PR DESCRIPTION
**Pull request by Nirbose**
# Description
Add ThrownExpBottle Entity

# Checklist
The following items should be checked before the pull request can be merged.
- [x] Code follows existing style.
- [x] Unit tests added (if applicable).

### CODE REVIEW

File: src/test/java/be/seeseemelk/mockbukkit/entity/ThrownExpBottleMockTest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+		assertEquals(Material.EXPERIENCE_BOTTLE, bottle.getItem().getType());
+	}
+
+	@Test
+	void testSetItem()
+	{
+		bottle.setItem(new ItemStack(Material.DIAMOND));
+
+		assertEquals(Material.DIAMOND, bottle.getItem().getType());
+	}
```
Comment (thelooter) [inline src/test/java/be/seeseemelk/mockbukkit/entity/ThrownExpBottleMockTest.java]: Please also test that the amount is always one, no matter the used ItemStack's amount

**Review by thelooter — CHANGES_REQUESTED** [review]
One little change

File: src/test/java/be/seeseemelk/mockbukkit/entity/ThrownExpBottleMockTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import org.bukkit.inventory.ItemStack;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ThrownExpBottleMockTest
```
Comment (thelooter) [inline src/test/java/be/seeseemelk/mockbukkit/entity/ThrownExpBottleMockTest.java]:
```suggestion
class ThrownExpBottleMockTest
```

Comment (Nirbose) [reply src/test/java/be/seeseemelk/mockbukkit/entity/ThrownExpBottleMockTest.java]: Done in the commit [b52e778](https://github.com/MockBukkit/MockBukkit/pull/913/commits/b52e77896fb4860405828817eb0a62da8f82a1ce)

File: src/test/java/be/seeseemelk/mockbukkit/entity/ThrownExpBottleMockTest.java (hunk 1 above)
Comment (Nirbose) [reply src/test/java/be/seeseemelk/mockbukkit/entity/ThrownExpBottleMockTest.java]: Done in the commit [b52e778](https://github.com/MockBukkit/MockBukkit/pull/913/commits/b52e77896fb4860405828817eb0a62da8f82a1ce)

**Review by thelooter — APPROVED** [review]
LGTM

**Pull request merged**